content/_en/partners/impact-stories.md

```diff
@@ -2,7 +2,7 @@
 layout: partners/impact-stories
 permalink: /partners/impact-stories/
 title: >-
-    # Impact stories
+    Impact stories
```

Comment:
Uhhhh why'd this happen? This messes up the titles on all the pages.

Reply:
<img width="751" alt="Screen Shot 2021-12-28 at 1 52 51 PM" src="https://user-images.githubusercontent.com/25647797/147597858-a83f092a-b1f4-4ead-aa5f-ffb77d967521.png">